lib/protoflow-blocks/doc/io/decode_hex.seq.mmd

```diff
@@ -0,0 +1,21 @@
+sequenceDiagram
+    autonumber
+    participant BlockA as Another block
+    participant DecodeHex.input as DecodeHex.input port
+    participant DecodeHex as DecodeHex block
+    participant DecodeHex.output as DecodeHex.output port
+    participant BlockB as Another block
+
+    BlockA-->>DecodeHex: Connect
+    DecodeHex-->>BlockB: Connect
+
+    loop DecodeHex process
+        BlockA->>DecodeHex: Message (Hexadecimal Bytes)
```

Comment:
Let's keep the symmetry with [`EncodeHex`](https://docs.rs/protoflow-blocks/latest/protoflow_blocks/io/struct.EncodeHex.html): this should just say "(Bytes)", also because "(Hexadecimal Bytes)" isn't a familiar concept

Reply:
done